js/ui/file_manager/ui.file_manager.common.js

```diff
@@ -22,34 +30,31 @@ const whenSome = function(arg, onSuccess, onError) {
     };
 
     if(!Array.isArray(arg)) {
-        return when(arg)
-            .then(onSuccess,
-                error => {
-                    if(error) {
-                        onError(createErrorInfo(0, error));
-                    }
-                });
+        arg = [ arg ];
     }
 
     const deferreds = arg.map((item, index) => {
         return when(item)
-            .then(result => createResult(result, true),
+            .then(
+                result => {
+                    const resultObj = createResult(result, true);
+                    if(resultObj.error && onError) {
```

Comment:
`resultObj.error` always  is null so the condition also always is null. Can we remove it?